src/widget_grid.js

```diff
@@ -119,9 +121,9 @@ const AgGridView = widgets.DOMWidgetView.extend({
             // remove all previous css rules
             var i = sheet.rules.length
             while (i--) {
-                if (sheet.rules[i].selectorText.startsWith(`#widget-grid-${this._id}`)) { 
+                if (sheet.rules[i].selectorText.startsWith(`#widget-grid-${this._id}`)) {
                     sheet.removeRule(i);
-                } 
+                }
             }
             // Add custom CSS
             this.model.get('css_rules_down').forEach(rule => {
```

Comment:
Let's try to:
1. Move all backend to this repository and the frontend will be inside a "js" folder. Make sure you can install the frontend and the backend from this repo
2. Add the backend change to support the scrolling.

Reply:
1. no luck
2. done in another repo